migrations/frontend/1528395945_make_batch_change_last_applied_at_nullable.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+ALTER TABLE IF EXISTS batch_changes
+ALTER COLUMN last_applied_at
+DROP NOT NULL;
+
+ALTER TABLE IF EXISTS batch_changes
+ALTER COLUMN last_applied_at
+DROP DEFAULT;
```

Comment:
Nitpick: I'd personally try to reduce the amount of text to make it easier to spot what the important bit is and combine those two statements

```suggestion
ALTER TABLE IF EXISTS batch_changes
  ALTER COLUMN last_applied_at DROP NOT NULL,
  ALTER COLUMN last_applied_at DROP DEFAULT;
```

Reply:
Great suggestion, thank you!